src/encoded/static/components/search.js

```diff
@@ -1436,108 +664,75 @@ TextFilter.propTypes = {
 };
 
 
-/**
- * Determine whether a facet should be displayed or not.
- * @param {object} facet One facet object from search results
- * @param {object} session From <App> React context
- * @param {object} sessionProperties From <App> React context
- *
- * @return True if given facet should be hidden
- */
-const isFacetHidden = (facet, session, sessionProperties) => (
-    facet.field === 'internal_status' && !sessionProperties.admin
-);
-
-
 // Displays the entire list of facets. It contains a number of <Facet> cmoponents.
-export const FacetList = (props, reactContext) => {
-    const { context, facets, filters, mode, orientation, hideTextFilter, addClasses, docTypeTitleSuffix } = props;
-
-    // Get "normal" facets, meaning non-audit facets.
-    const normalFacets = facets.filter(facet => facet.field.substring(0, 6) !== 'audit.');
-
-    let width = 'inherit';
-    if (facets.length === 0 && mode !== 'picker') return <div />;
-    let hideTypes;
-    if (mode === 'picker') {
-        // The edit forms item picker (search results in an edit item) shows the Types facet.
-        hideTypes = false;
-    } else {
-        // Hide the types facet if one "type=" term exists in the URL. and it's not the only
-        // facet.
-        hideTypes = filters.filter(filter => filter.field === 'type').length === 1 && normalFacets.length > 1;
-    }
-    if (orientation === 'horizontal') {
-        width = `${100 / facets.length}%`;
+export const FacetList = (props) => {
+    const { context, facets, filters, mode, orientation, hideTextFilter, addClasses, docTypeTitleSuffix, supressTitle, onFilter } = props;
+    if (facets.length === 0 && mode !== 'picker') {
+        return <div />;
     }
 
-    // See if we need the Clear Filters link or not. context.clear_filters.
-    let clearButton;
-    const searchQuery = context && context['@id'] && url.parse(context['@id']).search;
-    if (searchQuery) {
-        // Convert search query string to a query object for easy parsing.
-        const terms = queryString.parse(searchQuery);
+    const parsedUrl = context && context['@id'] && url.parse(context['@id']);
 
-        // See if there are terms in the query string aside from `searchTerm` or `advancedQuery`. We have a Clear
-        // Filters button if we do.
-        let nonPersistentTerms = _(Object.keys(terms)).any(term => term !== 'searchTerm' && term !== 'advancedQuery');
-        clearButton = nonPersistentTerms && (terms.searchTerm || terms.advancedQuery);
+    // See if we need the Clear Filters link or not. context.clear_filters.
```

Comment:
Could this be clarified? Maybe "see if we need the Clear filters link by checking context.clear_filters" or something?